server/meta.jsx

```diff
@@ -74,8 +74,13 @@ function isSDKUrl(hostname : string) : boolean {
     return false;
 }
 
+function isLocalUrl(host : string) : boolean {
+    const localUrls = [HOST.LOCALHOST_8000, HOST.LOCALHOST_8443, HOST.LOCALTUNNEL];
+    return localUrls.some(url => host === url);
```

Comment:
To be safe, could we also add an environment check here?
```suggestion
    return __ENV__ === ENV.LOCAL && localUrls.some(url => host === url);
```

Reply:
@gregjopa We could but this cannot occur in production.  The config.json will cause this to fail due to the CSP policy.  But don't mind adding for extra security.

Reply:
@gregjopa __ENV__ is not available on the server.  I think we're good here.  @bluepnume ?

Reply:
Can we check `process.env.NODE_ENV`? I think this is set by most paypal apps. May need to add a log to check if the correct value is `localhost` or `development` or something